File: Python-DSA/LargestEleArray.py
"""

find the largest element in the array 
Time complexity: O(n)

"""

arr =  [ 5, 3, 7, 2, 4 ] 

largest = arr[0]

for i in arr:
    if i > largest:
        largest = i

print(largest)

# other way

largestEle = max(arr)
print(largestEle)

File: JAVA-DSA/LargestEleArray.java
/*
 find the largest element in array 
 Time complexity : O(n)
*/

class LargestEleArray {
    public static void main(String args[]) {
        int n = 5;
        int arr[] = { 5, 3, 7, 2, 4 };

        int largest = arr[0];
        for ( int i = 0 ; i < n ; i++ ) {
            if ( arr[i] > largest ) {
                largest = arr[i];
            }
        }

        System.out.println("largest element --> " + largest);
    }
}